pom.xml

```diff
@@ -389,6 +389,9 @@
             <id>enforce</id>
             <configuration>
               <rules>
+                <requireJavaVersion>
+                  <version>1.8</version>
+                </requireJavaVersion>
```

Comment:
can we add same to the APIs, just with `17`?

Reply:
Yes, that would be the idea. This is the PR for the main branch, we'd merge this down to v2 and do another PR to add similar configuration to `apis`